common/common-javassist/src/test/java/io/servicecomb/common/javassist/TestJavassistUtils.java

```diff
@@ -86,7 +80,6 @@ public void testAddParameter() {
     String intfName = "cse.ut.TestAddParameter";
     classConfig.setClassName(intfName);
 
-    // List<String> method(Map<String, String> map, Set<String> set)
```

Comment:
this comments is what the target method will be.

Reply:
here, we create a new class in memory, and add a new method in the class
if we did not comments the targt method signature, it's hard to know what's the purpose

so i thinks, comments can be better, but delete it is not good.

Reply:
Is it reasonable if we define another method to hold the structure or use test case to clarify it? I think it is more understandable and believable. 
Or should we add more detailed comments to mention that rather than just a method definition?